src/main/java/com/yevhenberladyniuk/debts/service/DebtServiceImpl.java

```diff
@@ -0,0 +1,67 @@
+package com.yevhenberladyniuk.debts.service;
+
+import com.yevhenberladyniuk.debts.domain.Debt;
+import com.yevhenberladyniuk.debts.domain.Partner;
+import com.yevhenberladyniuk.debts.dto.CreateDebt;
+import com.yevhenberladyniuk.debts.repository.DebtRepository;
+import com.yevhenberladyniuk.debts.repository.PartnerRepository;
+import org.hibernate.validator.internal.util.privilegedactions.LoadClass;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class DebtServiceImpl implements DebtService{
+
+    DebtRepository debtRepository;
+
+    PartnerRepository partnerRepository;
+
+    @Autowired
+    public DebtServiceImpl(DebtRepository debtRepository, PartnerRepository partnerRepository) {
+        this.debtRepository = debtRepository;
+        this.partnerRepository = partnerRepository;
+    }
+
+    @Override
+    public List<Debt> findAllByPartner(Partner partner) {
+        return debtRepository.findAllByPartnerId(partner.getId());
+    }
+
+    @Override
+    public void create(CreateDebt createDebt, Partner partner) {
```

Comment:
As you have more than one `repository.save()`, mark this method  `@Transactional`